Source/SwiftLintFramework/Rules/Style/IdentifierNameRule.swift

```diff
@@ -38,6 +38,13 @@ struct IdentifierNameRule: ASTRule, ConfigurationProviderRule {
                 return []
             }
 
+            guard !configuration.excludedRegularExpressions.contains(where: {
+                guard let regex = try? NSRegularExpression(pattern: $0) else { return false }
```

Comment:
Creating the regex should already be done when parsing the configuration so that it only happens once and not every time an identifier is checked.

Reply:
Thank you for your review and I totally agree with your idea to consider `excluded` list as regular expressions :)

I pushed some commits applying the changes, but I'm not pretty sure if interpreting `excluded` list as regex in this way would be clear to other people too. ( By adding `^` to the beginning and `$` to the end )

Reply:
I cannot think of an example in which this would cause a problem. Up to now, only identifiers are supported. Wrapping them now into `^` and `$` and interpreting them as regular expressions should not cause any trouble even people are not aware of this.